src/models/event.ts

```diff
@@ -710,6 +711,10 @@ export class MatrixEvent extends TypedEventEmitter<MatrixEventEmittedEvents, Mat
         return this.clearEvent?.content?.msgtype === "m.bad.encrypted";
     }
 
+    public isEncryptedDisabledForUnverifiedDevices(): boolean {
```

Comment:
This should be a getter rather than a function

Reply:
I used the same behaviour than `isDecryptionFailure` which is also a function, it should be also a getter in this case ?

Reply:
The SDK predates getters being available in the language, we should be using them going forward though, not breaking backwards compatibility by mutating existing contracts